lya/analysis.py

```diff
@@ -53,3 +55,10 @@ def get_best_fit():
     pars : array-like
         Best-fit [alpha, beta, gamma] parameters 
     """
+
+    data = get_data_transfer_function()
+    f = lambda pars: objective(pars, data)
+    res = minimize(f, (0, 7, 0))
```

Comment:
I like it-- another way to do this without needing to invoke a `lambda` would be to pass the `args=` keyword; i.e.:
```python
res = minimize(objective, (0, 7, 0), args=(data,))
```